src/parameters.coffee

```diff
@@ -216,13 +216,18 @@ class LayerParam extends Param
   # @private
   value: ()->
     layerOptions = @origValue
+    
     if cloudinary.Util.isPlainObject(layerOptions)
       if layerOptions.resource_type == "text" || layerOptions.text?
         result = new cloudinary.TextLayer(layerOptions).toString()
       else if layerOptions.resource_type == "subtitles"
         result = new cloudinary.SubtitlesLayer(layerOptions).toString()
+      else if layerOptions.resource_type == "fetch"
+        result = new cloudinary.Layer(layerOptions).toString()
       else
         result = new cloudinary.Layer(layerOptions).toString()
+    else if Util.isString(layerOptions) && (layerOptions.search(/fetch:/) == 0)
+      result = "fetch:#{btoa(layerOptions.substr(6))}"
```

Comment:
`btoa` is a method of `Window` and will not be supported outside the browser. Please replace with a Pure JS implementation.